tcms/core/tests/test_admin.py

```diff
@@ -130,7 +130,8 @@ def test_regular_user_should_not_delete_another_user(self):
             username=self.regular_user.username, password="password"
         )
         response = self.client.get(
-            reverse("admin:auth_user_delete", args=[self.superuser.pk])
+            reverse("admin:auth_user_delete", args=[self.superuser.pk]),
+            follow=True,
```

Comment:
This is wrong! `follow=True` will send the current user to the list-view page, where they can see all users. And because they don't have permissions to do so there will be a 403 response which will cause the assertion below to report PASS. 

- Remove the `follow=True` line like it was before
- don't delete `self.superuser.pk`, delete another regular user account